.github/workflows/cd.yml

```
@@ -1,115 +1,80 @@
-      # 6. Docker 이미지 빌드 및 푸시
-      - name: Build and push Docker image
-        uses: docker/build-push-action@v4
-        with:
-          context: .
-          platforms: linux/amd64,linux/arm64
-          push: true
-          # 이미지 태그: {DockerHub유저명}/{리포지토리명}:{브랜치명}-{commit해시}
-          tags: ${{ env.IMAGE_NAME }}
+      - name: Build and push to Docker Hub
+        id: build-image
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:4:17: Double quote to prevent globbing and word splitting

(shellcheck)